Casks/m/morgen.rb

```diff
@@ -14,8 +14,13 @@
   livecheck do
     url "https://download.todesktop.com/210203cqcj00tw1/latest-mac.yml"
     regex(/Morgen\sv?(\d+(?:\.\d+)+).*?(?:Build\s)([a-z0-9]+)[._-]#{arch}\.dmg/i)
```

Comment:
```suggestion
    regex(/Morgen\s+v?(\d+(?:\.\d+)+).*?(?:Build\s+)([a-z0-9]+)[._-]#{arch}\.dmg/i)
```

Just some straightforward tweaks while I'm looking at this (i.e., we use `\s+` when matching a required space, as it will handle the expected space and also unexpected additional spaces due to typos, etc.).